packages/botonic-plugin-knowledge-bases/src/hubtype-knowledge-api-service.ts

```diff
@@ -14,28 +16,33 @@ export class HubtypeApiService {
 
   async inference(
     authToken: string,
-    question: string,
-    sources: string[]
+    sources: string[],
+    instructions: string,
+    messageId: string,
+    memoryLength: number
   ): Promise<
     AxiosResponse<{
```

Comment:
create an interface for `InferenceResponse` so it will be easier for us to maintain it

Reply:
maybe add it to `src/types`

Reply:
But this is not the object that the plugin resolves since the attributes are in snake_case as the backend resolves them. In the types file there is the KnowledgeBaseResponse interface with the attributes in camelCase which is the object that the call is debugged when the plugin is used and the attributes that are used in the bot